### DISCUSSION TITLE
Copy last response by on cmd + c

### DISCUSSION THREAD

**Discussion by japanese-goblinn** [intellibar/main#19] [Ideas]
If nothing is selected copy last response by pressing `cmd + c`

**Comment by astoilkov** [comment]
Hmm, this should already work in 0.8. Doesn't it work for you?

**Reply by japanese-goblinn** [reply]
Oh, I'm sorry! It actually works! I just didn't see any indication that it was copied. When I press copy as text, a tiny pop-up appears, but not when I press cmd + c.

Also, is it possible to omit the > that is inserted in the response text?

**Reply by astoilkov** [reply]
> Oh, I'm sorry! It actually works! I just didn't see any indication that it was copied. When I press copy as text, a tiny pop-up appears, but not when I press cmd + c.

Aha, yes. I'm writing it down. It's a good improvement.

> Also, is it possible to omit the > that is inserted in the response text?

Sorry. I'm not sure I understood this. Can you clarify a bit?

**Reply by japanese-goblinn** [reply]
> Sorry. I'm not sure I understood this. Can you clarify a bit?

Of course! I just figured out that the problem is not what I initially thought.

This `>` is a quoting in response, but it's not rendered as a quote in app and copied png. Here's an example
![image](https://github.com/intellibar/main/assets/33631005/fdda1e3e-405e-4e9f-ba5a-a59ed05a9ae6)

This is what i get if i copy the response 
```
Sure! Here's the revised text:

> Additionally, is it feasible to exclude the ">" symbol that is added to the response text?
```

**Reply by astoilkov** [reply]
Aha. Got it. Thanks. Yeah, makes sense. I will take a look at this.
